src/chainstate/stacks/db/blocks.rs

```diff
@@ -4886,21 +4886,41 @@ impl StacksChainState {
                             receipts.append(&mut clarity_tx.block.initialize_epoch_2_1()?);
                             applied = true;
                         }
+                        StacksEpochId::Epoch22 => {
```

Comment:
This `match` branch is getting a bit unruly with all these epochs.  It might make sense to (later) add a function called `initialize_epochs(cur_epoch: StacksEpochId, target_epoch: StacksEpochId)` or some such that carries out all the epoch initialization logic between `cur_epoch` and `target_epoch`.  Then we won't need to duplicate all this initialization code for each epoch.

Reply:
Yep, like the refactoring above, I'll think about if this makes sense for Epoch 2.3, and if not, it should be a non-consensus-changing refactor anyways.